vpr/src/power/PowerSpicedComponent.cpp

```diff
@@ -84,7 +85,7 @@ PowerCallibSize* PowerCallibInputs::get_entry_bound(bool lower,
 
 PowerSpicedComponent::PowerSpicedComponent(std::string component_name,
                                            float (*usage_fn)(int num_inputs, float transistor_size)) {
-    name = component_name;
+    name = std::move(component_name);
```

Comment:
instead of calling std::move here, we should pass a `const std::string &` as parameter, and just use assignment here.